web/html/src/manager/virtualization/nets/virtualization-nets-list-refresh-api.js

```diff
@@ -51,9 +52,9 @@ class VirtualizationNetsListRefreshApi extends React.Component<Props, State> {
           errors: [],
         });
       })
-      .catch((response) => {
+      .catch(jqXHR => {
         this.setState({
-          errors: [Network.errorMessageByStatus(response.status)],
+          errors: [Network.responseErrorMessage(jqXHR)]
```

Comment:
What is that `jqXHR` thing? I have no clue about that: I trust that you know more those things than I do

Reply:
It's the jQuery XMLHTTP request object.